client/verta/verta/client.py

```diff
@@ -180,15 +180,16 @@ def __init__(self, host=None, port=None, email=None, dev_key=None,
                 response = _utils.make_request("GET",
                                                "{}://{}/api/v1/modeldb/project/verifyConnection".format(conn.scheme, conn.socket),
                                                conn)
-            except requests.ConnectionError:
-                six.raise_from(requests.ConnectionError("connection failed; please check `host` and `port`"),
-                               None)
+            except requests.ConnectionError as err:
+                conn_error_msg = "connection failed; please check `host` and `port`; original error: \n{}".format(str(err))
+                six.raise_from(requests.ConnectionError(conn_error_msg), None)
```

Comment:
How do I induce this error?

Reply:
Actually I just realized I can just disconnect my internet. This is what it looks like:
```
 set host from environment
set email from environment
set developer key from environment
Traceback (most recent call last):
  File "../connection_err.py", line 2, in <module>
    Client()
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/client.py", line 185, in __init__
    six.raise_from(requests.ConnectionError(conn_error_msg), None)
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/external/six/__init__.py", line 737, in raise_from
    raise value
requests.exceptions.ConnectionError: connection failed; please check `host` and `port`; error message: 
HTTPSConnectionPool(host=... port=...): Max retries exceeded with url: /api/v1/modeldb/project/verifyConnection (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10a07fa90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
```

Reply:
Hmm that's not quite what I had in mind when writing the ticket.

- We should change this to use the original exception object, instead of creating a new `ConnectionError`, and simply modify its message and then proceed with raising it. [Here](https://github.com/VertaAI/modeldb/blob/545cfb9/client/verta/verta/_internal_utils/_utils.py#L533-L534)'s an example of the client doing something similar.
- I think ``"connection failed; please check `host` and `port`\n\n" + original_msg``would look nicer, though I'm not totally certain how it will end up looking. Could you give that a try?

Reply:
This is what I get:
```Python 3
set host from environment
set email from environment
set developer key from environment
Traceback (most recent call last):
  File "../connection_err.py", line 2, in <module>
    Client()
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/client.py", line 185, in __init__
    six.raise_from(err, None)
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/external/six/__init__.py", line 737, in raise_from
    raise value
requests.exceptions.ConnectionError: connection failed; please check `host` and `port`; error message: 

HTTPSConnectionPool(host=..., port=...): Max retries exceeded with url: /api/v1/modeldb/project/verifyConnection (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10a10aa90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
```
See [latest commit](https://github.com/VertaAI/modeldb/pull/1640/commits/d36093f61e311432d4175b37ffae4077a2df298e).